file_upload.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'json'
+require 'digest'
+
+# ================================================================
+# A Ruby example that demonstrates the workflow for uploading a
+# file to Sentera's cloud storage and then using the file with
+# Sentera's GraphQL API to associate it with a resource such as a
+# field, survey, feature set, etc.
+#
+# Full documentation of this workflow can be found here:
+# https://api.sentera.com/api/getting_started/uploading_files.html
+#
+# Contact support@sentera.com with any questions.
+# ================================================================
+
+GQL_ENDPOINT = 'https://api.sentera.com/graphql'
+
+# See https://api.sentera.com/api/getting_started/authentication_and_authorization.html
+# for details on how to obtain an auth token to use with Sentera's GraphQL API
+AUTH_TOKEN = '<Your auth token goes here>'
```

Comment:
This is a scary LOC for a public repo, because we don't want anyone accidentally publishing their token. How about committing a dummy `token.txt.example` to the repo and `.gitignore`ing `token.txt`, so that the scripts read the token from the filesystem and there is reduced chance one is leaked?

Reply:
I made this change. I complicates things a little bit for someone using the example, but I agree that it does protect our team from accidentally publishing a valid FieldAgent auth token.

https://github.com/SenteraLLC/file-upload-examples/pull/1/commits/d6431fba15b360f32596cb48aae399c778668e45

Reply:
Thanks, Denis. Yeah, definitely some additional indirection unfortunately.